packages/supi/test/storeUsages.ts

```diff
@@ -0,0 +1,37 @@
+import prepare from '@pnpm/prepare'
+import { FindPackageUsagesResponse } from '@pnpm/store-controller-types';
+import {
+  installPkgs,
+  storeUsages,
+} from 'supi'
+import tape = require('tape');
+import promisifyTape from 'tape-promise'
+import { testDefaults } from './utils'
+
+const test = promisifyTape(tape);
+
+test('find usages for newly installed package', async (t: tape.Test) => {
+  const project = prepare(t);
+
+  // Install deps
+  await installPkgs(['is-negative@2.1.0'], await testDefaults({ save: true }));
```

Comment:
This will have to be updated once I merge #1528

I did a big refactoring in supi.

Reply:
Ok, I'll take a look at those changes and update those soon.

Reply:
you can run `pnpm run lint -- --fix` in every project